all packit builds are failing

e.g. https://download.copr.fedorainfracloud.org/results/packit/bootc-dev-bcvk-320/centos-stream-10-x86_64/10871193-bcvk/builder-live.log.gz

```
  thread 'main' (1031) panicked at /builddir/build/BUILD/bcvk-0.18.0/vendor/aws-lc-sys/builder/cc_builder.rs:855:9:
  COMPILER: gccARGS: ["-O3", "-Wl,-z,relro", "-Wl,--as-needed", "-Wl,-z,pack-relative-relocs", "-Wl,-z,now", "-specs=/usr/lib/rpm/redhat/redhat-hardened-ld", "-specs=/usr/lib/rpm/redhat/redhat-hardened-ld-errors", "-specs=/usr/lib/rpm/redhat/redhat-annobin-cc1", "-Wl,--build-id=sha1", "-specs=/usr/lib/rpm/redhat/redhat-package-notes", "/builddir/build/BUILD/bcvk-0.18.0/vendor/aws-lc-sys/aws-lc/tests/compiler_features_tests/memcmp_invalid_stripped_check.c", "-Wno-unused-parameter", "-o", "/builddir/build/BUILD/bcvk-0.18.0/target/rpm/build/aws-lc-sys-0bf3f0db01ff613d/out/memcmp_invalid_stripped_check"]EXECUTED: trueERROR: /usr/bin/ld: /tmp/ccX5lcWd.o: relocation R_X86_64_32 against `.rodata' can not be used when making a PIE object; recompile with -fPIE
  /usr/bin/ld: failed to set dynamic section sizes: bad value
  collect2: error: ld returned 1 exit status
  OUTPUT: Failed to compile memcmp_invalid_stripped_check           
```

--- a/crates/integration-tests/src/tests/run_ephemeral_ssh.rs
+++ b/crates/integration-tests/src/tests/run_ephemeral_ssh.rs
@@ -311,7 +311,8 @@ fn test_run_ephemeral_ssh_broken_image_cleanup() -> TestResult {
 
     // Verify the error message indicates the problem
     assert!(
-        stderr.contains("Failed to read kernel modules directory")
+        stderr.contains("No kernel found")
+            || stderr.contains("Failed to read kernel modules directory")
             || stderr.contains("Container exited before SSH became available")
             || stderr.contains("Monitor process exited unexpectedly"),
         "Expected error about missing kernel or container failure, got: {}",

--- a/crates/kit/Cargo.toml
+++ b/crates/kit/Cargo.toml
@@ -25,7 +25,7 @@ tracing = { workspace = true }
 tracing-subscriber = { workspace = true }
 tracing-error = { workspace = true }
 shlex = "2"
-reqwest = { version = "0.13", features = ["blocking"] }
+reqwest = { version = "0.13", default-features = false, features = ["blocking", "native-tls", "charset", "http2", "system-proxy"] }
 tempfile = "3"
 uuid = { version = "1.10", features = ["v4"] }
 xshell = { workspace = true }

